### DISCUSSION TITLE
Give more ideas !!!

### DISCUSSION THREAD

**Discussion by vikhyatsingh123** [vikhyatsingh123/Naruto-Shippuden#1033] [Ideas]
You are most welcome if anyone wants to give more ideas to add more features or want to improve this website then please let me know.

**Comment by ghost** [comment]
Hi everyone.

I want to help, but I want to know if this project is active. *Is this project active?*

**Comment by vikhyatsingh123** [comment]
Yes sure

**Reply by ghost** [reply]
Hi vikhyatsingh123.

> Yes sure

thank you for feedback.

**Reply by Charcrede** [reply]
i download your naruto's website, and i can say you it is just perfect, it just lake a lot of information . And i wanted to ask you something then, how have you cursor become a kunaï when hover ?

**Reply by vikhyatsingh123** [reply]
@Charcrede Sorry for the late reply, I saw your message today. You can find how we implemented kunai pointer on hover in style.css file

**Comment by ghost** [comment]
Hello everyone!

*what do you all think about the idea of ​​creating an api for the Naruto-Shippuden website?*

`https://vikhyatsingh123.github.io/Naruto-Shippuden/api/naruto ->`
```json
{
"naruto": "Naruto Uzumaki is the main protagonist in the popular manga and anime series Naruto. He is a cheerful, hyperactive, strong-willed, and occasionally simple-minded young shinobi from the village of Konoha (or Leaf Village). Since Naruto has the Nine Tails Fox sealed inside him, he is able to use the Fox's chakra, which is much greater than the average human. Initially Naruto and the Fox hated each other, and would rarely grant Naruto his power unless they were going to die. Eventually, they become friends, and Naruto then refers to the fox by his name, Kurama. At this point, Naruto can use Kurama's chakra at will."
}
```
